### ISSUE TITLE
Loss calculation

### ISSUE THREAD

**Issue by osivaz61** [sharonchokuwa/VAE-DG#1]
You are using F.binary_cross_entropy(recon_x, x, reduction='sum')  function for loss calculation. But the value interval of the preprocessed image -2 to 2. But the recon_x is 0 to 1. Is this right?
